docs/book.toml

```diff
@@ -2,10 +2,12 @@
 authors = ["zimbatm"]
 language = "en"
 multilingual = false
-src = "."
+src = "src"
```

Comment:
What is the motivation for moving this one level down? It's one more level to traverse when navigating the code.

Reply:
This structure is described in the [mdbook docs](https://rust-lang.github.io/mdBook/guide/creating.html#source-files)